.conda/build.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
 
-# Install pymanopt and tensorflow_probability via pip because there isn't a conda package
+# Install pymanopt and tensorflow_probability via pip because there isn't a conda 
+# package, and install tensorflow 2.3 from pip because tensorflow_probability
+# requires it and latest in conda is 2.2. 
+PIP_NO_INDEX=False $PYTHON -m pip install tensorflow>=2.3
```

Comment:
You need to quote or escape the `>`, otherwise Bash interprets it as a redirect, e.g., `tensorflow\>=2.3`. We cannot see in the Travis logs what happened to the installation because of this redirect.

Reply:
Ah, thanks. I blame my fancy programming font ligature.

Reply:
I don't think that was it though? Still fails.

Reply:
Thanks for trying. Not sure what is happening. I'll experiment with building the Conda package manually to understand. If I don't figure it out in the next few days, I plan to merge the PR nevertheless and solve the build issue later.

Reply:
Sounds good! I'll get rolling on the next piece (MN-SRM) and PR that when it's ready.